projboard/tests/test_client.py

```diff
@@ -20,15 +21,35 @@ def test_homepage(client, articles_num):
     assert 'landing/homepage.html' in template_names
 
 
-def test_get_searchpage(client):
-    response = client.get("/search/")
+@pytest.fixture
+@pytest.mark.django_db
+def User2(client):
```

Comment:
client is not needed here, it is a fixture
You have fixtures already for creating a user
If you need to parse a parameter please fix the fixture 

Please refer this 
https://github.com/redhat-beyond/Smarticle/blob/main/projboard/tests/conftest.py#L11

Reply:
done